### DISCUSSION TITLE
[Feature] Add checkbox to include/exclude nullable states

### DISCUSSION THREAD

**Discussion by rvzug** [malzariey/filament-daterangepicker-filter#110] [Ideas]
Wouldn't it be nice if you also can choose through a checkbox if you want to include/exclude dates with a null value?

So, please give me the items in range x-z, and null values.

could be placed here?
![image](https://github.com/user-attachments/assets/255088ba-29da-433f-b2a4-a29c3dbab784)

**Comment by malzariey** [comment]
You could just clear the DateRangeFilter By pressing on the calendar icon. Which should be sufficient for now. Might as well add it on the pciker itself. 

Thanks for your suggestion

**Reply by rvzug** [reply]
For me the suggestion is: the case where you want to filter on: ```date-between x and y``` including or excluding ```date-is-null```.
So not only ```date-between``` **or** ```null```.

"Show me items with due date between x and y, or no due date"

**Reply by malzariey** [reply]
Ohh, It would be possible. 

The option label would need to be changed depending on column (needs to be short) and needs abit of work to override daterangepicker (Below Ranges). 

Intrested in a PR?

**Reply by rvzug** [reply]
I'm not that deep into js. I did look into it, but I think I get lost in ```resources/js/filament-daterangepicker.js```. Also this would only apply for the filter, not the date-selector as form-field. I don't know if that is impacting the package?

![image](https://github.com/user-attachments/assets/13e2de4d-f2e5-4d6b-aa3f-bffa8316f855)

**Reply by malzariey** [reply]
In the new version its nullable by default
